src/App.js

```diff
@@ -1,29 +1,41 @@
-import React, { Component } from 'react';
-import './App.css';
-import openSocket from 'socket.io-client';
+import React, { Component } from "react";
+import "./App.css";
+import openSocket from "socket.io-client";
 
-const socket = openSocket('http://localhost:8000');
+const socket = openSocket("http://localhost:8000");
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      username: '',
-      value: '',
-      messages: []
+      username: "",
+      value: "",
+      messages: [],
+      isTyping: false,
```

Comment:
Tror inte denna borde behövas. Du använder ju ändå `this.state.typingUser !== ""` för att skriva ut vem som skriver.

Reply:
Fixat